src/pyscipopt/recipes/primal_dual_evolution.py

```diff
@@ -0,0 +1,50 @@
+from pyscipopt import Model, Eventhdlr, SCIP_EVENTTYPE, Eventhdlr
+
+def get_primal_dual_evolution(model: Model):
+
```

Comment:
I think we need a good docstring here. Describing most importantly where the data is saved and how one would read it

Reply:
What do you think of this, Mo?

>     """
    Attaches an event handler to a given SCIP model that collects primal and dual solutions,
    along with the solving time when they were found.
    The data is saved in model.data["primal_log"] and model.data["dual_log"]. They consist of
    a list of tuples, each tuple containing the solving time and the corresponding solution.
    A usage example can be found in examples/finished/plot_primal_dual_evolution.py. The
    example takes the information provided by this recipe and uses it to plot the evolution
    of the dual and primal bounds over time. 
    """